airflow/dag_processing/manager.py

```diff
@@ -177,18 +178,17 @@ def _run_processor_manager(
         # Make this process start as a new process group - that makes it easy
         # to kill all sub-process of this at the OS-level, rather than having
         # to iterate the child processes
+
         set_new_process_group()
-        span = Trace.get_current_span()
-        span.set_attribute("dag_directory", str(dag_directory))
         setproctitle("airflow scheduler -- DagFileProcessorManager")
         reload_configuration_for_dag_processing()
         processor_manager = DagFileProcessorManager(
             dag_directory=dag_directory,
             max_runs=max_runs,
-            processor_timeout=processor_timeout,
+            processor_timeout=processor_timeout.total_seconds(),
             signal_conn=signal_conn,
         )
-        processor_manager.start()
+        processor_manager.run()
```

Comment:
We will also need to change:

https://github.com/apache/airflow/blob/8848d6e35f6ead2cc1ed9887af8360fbd61c78c3/airflow/jobs/dag_processor_job_runner.py#L62